aecore_eqc/txs_eqc.erl

```diff
@@ -1434,6 +1437,21 @@ prop_txs(Fork) ->
                                            {total, Total == 0 orelse equals(Total, ?PatronAmount - FeeTotal)}]))))))))
     end)))).
 
+load_app_if_unloaded(App) ->
```

Comment:
far too verbose.... this is test code. Just load it no optimizations

Reply:
Ok - I will make load less verbose.